src/components/enter-password/enter-password-controller.ts

```diff
@@ -45,7 +51,7 @@ export function enterPasswordPost(
       "password",
       req.t("pages.enterPassword.password.validationError.incorrectPassword")
     );
-
+    //TODO this won't work properly
     renderBadRequest(res, req, TEMPLATE_NAME, error);
```

Comment:
has this been addressed? the todo

Reply:
Fixed, validation works for both enter password pages.